msal/src/androidTest/java/com/microsoft/identity/client/PublicClientConfigurationTest.java

```diff
@@ -29,20 +29,19 @@
 import androidx.test.ext.junit.runners.AndroidJUnit4;
 
 import com.microsoft.identity.client.configuration.HttpConfiguration;
-import com.microsoft.identity.common.internal.authorities.AccountsInOneOrganization;
-import com.microsoft.identity.common.internal.authorities.ActiveDirectoryFederationServicesAuthority;
-import com.microsoft.identity.common.internal.authorities.AllAccounts;
-import com.microsoft.identity.common.internal.authorities.AnyOrganizationalAccount;
-import com.microsoft.identity.common.internal.authorities.AnyPersonalAccount;
-import com.microsoft.identity.common.internal.authorities.Authority;
-import com.microsoft.identity.common.internal.authorities.AzureActiveDirectoryAuthority;
-import com.microsoft.identity.common.internal.authorities.AzureActiveDirectoryB2CAuthority;
-import com.microsoft.identity.common.internal.ui.AuthorizationAgent;
+import com.microsoft.identity.common.java.authorities.AccountsInOneOrganization;
```

Comment:
Just noticed that it looks like we've done away with the internal package names.  How are we clarifying for consumers of the code what's being described by semantic versioning of a library vs.  what is not being addressed by semantic versioning?

Reply:
As mentioned in the Teams chat, let's discuss about this in the next team meeting.